services/app-api/handlers/reports/archive.ts

```diff
@@ -15,14 +15,19 @@ export const archiveReport = handler(async (event, context) => {
   if (getCurrentReport?.body) {
     const currentReport = JSON.parse(getCurrentReport.body);
     const currentArchivedStatus = currentReport?.archived;
+    const reportType = currentReport.reportType;
 
     // Delete raw data prior to updating
     delete currentReport.fieldData;
     delete currentReport.formTemplate;
 
     // toggle archived status in report metadata table
     const reportMetadataParams = {
-      TableName: process.env.MCPAR_REPORT_TABLE_NAME!,
+      // TODO: chain future reports here
+      TableName:
+        reportType === "MCPAR"
+          ? process.env.MCPAR_REPORT_TABLE_NAME!
+          : process.env.MLR_REPORT_TABLE_NAME!,
```

Comment:
Suggestion: I'd probably make a global object or enumerator to hold the report table names, like:
```
const reportTables = {
  MCPAR: process.env.MCPAR_REPORT_TABLE_NAME!,
  MLR: process.env.MLR_REPORT_TABLE_NAME!,  
  ...
}
```

Then you can just import that global constant and reference what you want from it as keys on an object, like:
`TableName: reportTables[reportType],`

Reply:
_Applies to all other similar areas with report tables and form buckets below_